src/themes/shared/slotRecipes/vehicleReference.ts

```diff
@@ -1,15 +1,14 @@
-import { ComponentStyleConfig } from '@chakra-ui/react';
+import { defineSlotRecipe } from '@chakra-ui/react';
 
-const parts = ['carTitle', 'price', 'dealerName', 'dealerAddress'];
 const wordBreak = 'break-word';
 
-const VehicleReference: ComponentStyleConfig = {
-  parts,
-  baseStyle: {
+export const VehicleReference = defineSlotRecipe({
+  slots: ['carTitle', 'price', 'dealerName', 'dealerAddress'],
+  base: {
```

Comment:
`base` styles are the styles applied to the default variant.
More variants can be defined in `variants` key. 
Docs on recipes can be found here: https://www.chakra-ui.com/docs/theming/customization/recipes